webpack.config.js

```diff
@@ -12,6 +12,18 @@ module.exports = {
       {
         test: /\.css$/,
         use: ['style-loader', 'css-loader']
+      },
```

Comment:
@siawo How is this loader chain working? The SVG extension seems to be checked in various loaders. Hows is this working, exactly?

Reply:
The loader chaining works in the reverse order, that is from right to left of an array rules.
The processed data from one loader is passed to the next next loader as input data.